backend/pkg/kubeconfig/kubeconfig.go

```diff
@@ -408,3 +411,11 @@ func LoadAndStoreKubeConfigs(kubeConfigStore ContextStore, kubeConfigs string, s
 
 	return nil
 }
+
+// makeDNSFriendly converts a string to a DNS-friendly format.
+func makeDNSFriendly(name string) string {
+	name = strings.ReplaceAll(name, "/", "-")
```

Comment:
Probably reducing the likelihood of a clash could be done with some less common marker? 

For example "--" would be less likely to clash than "-"?

"front end" -> "front-end"
"front end" -> "front--end"

In this example "front--end" is less likely to exist than "front-end".

Reply:
uI agree. Also, I think that having the same replacement for 2 different characters (`/` and ` `) is probably going to end up in problems to decode. Having spaces is unlikely to happen [AFAIU](https://stackoverflow.com/questions/74516648/eks-aks-cluster-name-convention), but I agree with keeping that as a defensive measure.
How about:
`my/cluster` -> `my--cluster`
`my cluster` -> `my__cluster` (double underscore)

I am assuming that double underscore and double dash are not used by any convention out there.
BTW, this all happens in memory, or are we storing the encoded cluster names somewhere? Because if we are storing them, the need for having something with minimal clash chances is of course very high.